services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/flows/BandwidthSpec.groovy

```diff
@@ -133,15 +154,40 @@ class BandwidthSpec extends BaseSpecification {
             }
         }
         flow.maximumBandwidth = involvedBandwidths.max() + 1
-        flow.ignoreBandwidth = false
         northboundService.addFlow(flow)
 
         then: "Flow is not created because flow path should not be found"
         def exc = thrown(HttpClientErrorException)
         exc.rawStatusCode == 404
+
+        when: "Create a flow with a valid bandwidth"
```

Comment:
Don't really see a reason why not to split this into two different tests.

Reply:
Will do.